### DISCUSSION TITLE
rAudio 20221117: GUI volume always returns to 100%

### DISCUSSION THREAD

**Discussion by princedurail** [rern/rAudio#1109] [Fixed Issues]
Hi rern,

Thank you for your great work, I hope to be clear in my explanations, I am French and use google translation.
 
since update 20221117 the GUI volume control systematically returns to 100% if I use my Hifiberry-amp card with the GUI.
there is no such problem when I select the headphone output or when I do not select "Volume" in the Playback menu and I use a rotary encoder to adjust the volume.
If I set the volume to zero in the GUI, it is indeed zero in Alsamixer master, then goes back to 100% in the GUI but not in Alsamixer, if I click on the GUI volume it goes to 100% in alsamixer , I can adjust the volume normally in GUI and alsamixer, then it goes back to 100% in GUI but not in Alsamixer.

I use a RPI 3B+ with Hifiberry Amp and Rotary encoder

Didier

**Comment by rern** [comment]
Hi Didier,

Please show output of command line: `amixer scontents`

**Comment by princedurail** [comment]
#  amixer scontents
Simple mixer control 'Master',0
  Capabilities: volume volume-joined
  Playback channels: Mono
  Capture channels: Mono
  Limits: 0 - 248
  Mono: 144 [58%]
Simple mixer control 'Channels',0
  Capabilities: volume
  Playback channels: Front Left - Front Right
  Capture channels: Front Left - Front Right
  Limits: 0 - 248
  Front Left: 200 [81%]
  Front Right: 200 [81%]

**Comment by princedurail** [comment]
Hi rern,
Problem solved with update 20221123.
thank
